satpy/tests/test_readers.py

```diff
@@ -64,6 +65,17 @@
 real_import = builtins.__import__
 
 
+@pytest.fixture
+def viirs_file(tmp_path, monkeypatch):
+    """Create a dummy viirs file."""
+    filename = 'SVI01_npp_d20120225_t1801245_e1802487_b01708_c20120226002130255476_noaa_ops.h5'
+
+    monkeypatch.chdir(tmp_path)
+    # touch the file so it exists on disk
+    open(filename, 'w').close()
+    return filename
```

Comment:
Is this different from
```python
open(tmp_path / filename, 'w').close()
return filename
```

If not, avoiding additional builtin fixtures would make the code easier to read.

Also, `tmp_path` isn't added to the `NOTE` at the top of the file :wink:

Reply:
Or rather `return tmp_path / filename`. Or does the reader need to have the file in `cwd`?

Reply:
oh, sorry I didn't see this before I merged...
Yes, the cwd needs to be set, because `find_files_and_readers` looks there by default, and that's what we want to test.

Also, I decide to make a fixture because we need both `monkeypatch` and `tmp_path`, so I tought I would be cleaner just to pass one fixture to the test functions.
I'll make a PR for the note.